spring-cloud-azure-starters/spring-cloud-starter-azure-config/README.md

```diff
@@ -73,16 +74,20 @@ Multiple labels can be separated with comma, if duplicate keys exists for multip
 ### Watch configuration change
 Watch feature allows the application to load the latest property value from configuration store automatically, without restarting the application.
 
-By default, the watch feature is enabled. It can be disabled with below configuration:
+By default, the watch feature is disabled. It can be enabled with below configuration:
 ```
-spring.cloud.azure.config.watch.enabled=false
+spring.cloud.azure.config.watch.enabled=true
 ```
 
-Change property key in the configuration store on Azure Portal, e.g., /application/config.message, log similar with below will be printed on the console.
+Change certain property key in the configuration store on Azure Portal, e.g., /application/config.message, log similar with below will be printed on the console.
 ```
 INFO 17496 --- [TaskScheduler-1] o.s.c.e.event.RefreshEventListener       : Refresh keys changed: [config.message]
 ```
 The application now will be using the updated properties. By default, `@ConfigurationProperties` annotated beans will be automatically refreshed. Use `@RefreshScope` on beans which are required to be refreshed when properties are changed.
+By default all the keys in the configuration store matching configured conditions will be watched, to watch certain key change only, configure the watch-key property for each configuration store.
```

Comment:
How about something like this (feel free to update)?

_By default, all the keys in a configuration store matching configured conditions will be watched. To prevent configuration changes are picked up in the middle of an update of multiple keys, you are recommended to use the **watched-key** property to watch a specific key that signals the completion of your update so all configuration changes can be refreshed together._

Reply:
Thanks, updated.